src/View/Helper/FormHelper.php

```diff
@@ -160,8 +161,8 @@ public function submit($caption = null, array $options = [])
             $class = array_unique(array_merge($class, $optionsClass));
             unset($options['class']);
         }
-
         $options = Hash::merge($options, ['class' => $class]);
+        $options['class'] = ButtonWidget::applyStyle($options['class']);
```

Comment:
OptionsAwareTrait is part of CakePHP.. That method sets the specific styles for bootstrap.